WTWSMS/localisation/WtWSMS_events.csv

```diff
@@ -1138,7 +1138,7 @@ imperial_legacy.49.name;[From.PrimaryTitle.GetName] disbanding;[From.PrimaryTitl
 imperial_legacy.49.desc;Without the authority of an imperial liege controlling their historical naval port, the [From.PrimaryTitle.GetName] has lost its roots and was forced to disbanded.;Sans l'autorit� d'un suzerain imp�rial contr�lant leur port d'attache historique, la [From.PrimaryTitle.GetName] a perdu ses racines et a �t� forc�e de se dissoudre.;;;;;;;;;;;;x
 ###### Historical events;;;;;;;;;;;;x
 historical_wtwsms.1.name;Restoration of Zeno;Restauration de Z�non;;;;;;;;;;x
-historical_wtwsms.1.desc;You are now once again the emperor in Constantinople. In 475, you had after a conspiracy lost the throne to the Miaphysite Basiliscus and been forced to flee the capital. Subsequently after loosing the throne and being defeated, Basiliscus has been imprisoned and safely awaits your decision in jail..;Vous �tes maintenant de nouveau l'empereur de Constantinople. En 475, apr�s un complot, vous aviez perdu le tr�ne au profit du Miaphysite Basiliscus et aviez �t� contraint de fuir la capitale. Apr�s avoir �t� vaincu et puis perdu le tr�ne, Basiliscus a d�sormais �t� emprisonn� et attend en toute s�curit� votre d�cision en prison...;;;;;;;;;;x
+historical_wtwsms.1.desc;You are now once again the emperor in Constantinople. In 475, you had after a conspiracy lost the throne to the Miaphysite Basiliscus and been forced to flee the capital. Subsequently after loosing the throne and being defeated, Basiliscus has been imprisoned and safely awaits your decision in prison...;Vous �tes maintenant de nouveau l'empereur de Constantinople. En 475, apr�s un complot, vous aviez perdu le tr�ne au profit du Miaphysite Basiliscus et aviez �t� contraint de fuir la capitale. Apr�s avoir �t� vaincu et puis perdu le tr�ne, Basiliscus a d�sormais �t� emprisonn� et attend en toute s�curit� votre d�cision en prison...;;;;;;;;;;x
```

Comment:
Any reason for jail -> prison? With prison and imprisoned the text is a bit repetitive, hence my usage of jail.

Reply:
"Prison" seems more "medieval" :P but that was my personal impression, I can revert it.

Reply:
No, we can use another synonym in that case. "dungeon" or "penitentiary" could be alternatives.

Reply:
Done, replaced with "dungeon".